src/slider/stories/MultiSlider.component.tsx

```diff
@@ -0,0 +1,159 @@
+   */
+  showTip?: boolean;
+  /**
+   * The `value` of the slider indicator.
+   *
+   * If `undefined`/`not valid` the slider bar will be the optimum of min & max
+   * @default [25,50,75]
+   */
+  values?: number[];
+  /**
+   * The minimum value of the slider
+   * @default 0
+   */
+  min?: number;
+  /**
+   * The maximum value of the slider
+   * @default 100
+   */
+  max?: number;
+  /**
+   * The step in which increments/decrements have to be made
+   * @default 1
+   */
+  step?: number;
+  /**
+   * If `true`, the slider will be disabled
+   * @default false
+   */
+  isDisabled?: boolean;
+  /**
+   * Orientation of the slider
+   * @default "horizontal"
+   */
+  orientation?: "horizontal" | "vertical";
+  /**
+   * Direction of the slider
+   * @default false
+   */
+  reversed?: boolean;
+  /**
+   * True, if the direction of the slider is reversed
+   * @default false
+   */
+  isReversed?: boolean;
+  /**
+   * Get the formated value based on number format options
+   */
+  formatOptions?: Intl.NumberFormatOptions;
+  /**
+   * Get the value when it changes
+   */
+  onChange?: (values: number[]) => void;
+  /**
+   * Get the value when dragging is started
+   */
+  onChangeEnd?: (value: number[]) => void;
+  /**
+   * Get the value when dragging is stopped
+   */
+  onChangeStart?: (value: number[]) => void;
+}
+
+export const App: React.FC<AppProps> = args => {
+  const { label, isReversed, origin: originProp, onChange, ...rest } = args;
+
+  const state = useSliderState({
+    value: [25, 50, 75],
+    reversed: isReversed,
+    ...rest,
+  });
+  const { values, getThumbValueLabel, getThumbPercent } = state;
+
+  const isVertical = args.orientation === "vertical";
+
+  React.useEffect(() => {
+    onChange?.(values);
+  }, [onChange, values]);
+
+  return (
+    <div
+      className="chakra-slider-group"
+      role="group"
+      aria-labelledby="styled-slider"
+    >
+      <div className="slider-label">
+        <label className="label" id="styled-slider">
+          {`${args.label ? args.label : "Styled"} Slider`}
+        </label>
+        <div className="value">{JSON.stringify(state.values)}</div>
+      </div>
+
+      <div className={`slider ${isVertical ? "vertical" : ""}`}>
+        <SliderTrack {...state} className="slider-track-container">
+          <div className="slider-track" />
+        </SliderTrack>
+
+        {[...new Array(values.length).keys()].map(index => {
+          return (
+            <div
+              className="slider-thumb"
+              key={`thumb-${index}`}
+              style={{
+                right:
+                  isReversed && `calc(${getThumbPercent(index) * 100}% - 7px)`,
+                left:
+                  !isReversed &&
+                  !isVertical &&
+                  `calc(${getThumbPercent(index) * 100}% - 7px)`,
+                bottom:
+                  isVertical && `calc(${getThumbPercent(index) * 100}% - 7px)`,
+              }}
```

Comment:
need to fix this in all the examples.

![image](https://user-images.githubusercontent.com/35374649/97986156-54b5d200-1dff-11eb-9293-9226f1c5e2de.png)